app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.example.drawingactivity">
 
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Are you sure you need this permission just to grab an image? I'm unsure if it's necessary

Reply:
Removed it